independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json

```diff
@@ -0,0 +1,89 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Default title",
+        "description": "",
+        "version": "1.0"
+    },
+    "host": "[YOURFUNCTIONAPP].azurewebsites.net",
+    "basePath": "/",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [],
+    "produces": [],
+    "paths": {
+        "/api/DynamicsAuthStart": {},
+        "/": {},
+        "/DynamicsAuth": {},
+        "/api/DynamicsAuth": {
+            "post": {
+                "responses": {
+                    "default": {
+                        "description": "default",
+                        "schema": {}
+                    }
+                },
+                "summary": "DynamicsAuth",
+                "operationId": "DynamicsAuth",
+                "description": "DynamicsAuth",
```

Comment:
@Stephan-Rijk I would highly recommend you having different values for summary and description. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).